src/reporting/interfaces/mappingInterfaces/CalculatedProperties.ts

```diff
@@ -0,0 +1,159 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Link, PagedResponseLinks } from "../Links";
+import { QuantityType } from "./GroupProperties";
+
+/**
+ * Defines a CalculatedProperty or 'column' for a Group. It is calculated for each element returned by the Group's query.
+ * @export
+ * @interface CalculatedProperty
+ */
+export interface CalculatedProperty {
+  /**
+   * The CalculatedProperty Id.
+   * @type {string}
+   * @memberof CalculatedProperty
+   */
+  id: string;
+  /**
+   * Name of the Property (OData v4 SimpleIdentifier).
+   * @type {string}
+   * @memberof CalculatedProperty
+   */
+  propertyName: string;
+  /**
+   * The type of the CalculatedProperty. One of 'Area', 'Length', 'Volume', 'BoundingBoxLongestEdgeLength', 'BoundingBoxIntermediateEdgeLength', 'BoundingBoxShortestEdgeLength', 'BoundingBoxDiagonalLength', 'BoundingBoxLongestFaceDiagonalLength', 'BoundingBoxIntermediateFaceDiagonalLength', 'BoundingBoxShortestFaceDiagonalLength'.
```

Comment:
There is no need to list all the possible values, when the value is an enum.

Reply:
removed all such comments